src/Skia/Avalonia.Skia/Helpers/LambdaDisposable.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+
+namespace Avalonia.Skia.Helpers
+{
+    /// <summary>
+    /// Executes a callback when dispose is called.
+    /// </summary>
+    internal class LambdaDisposable : IDisposable
```

Comment:
Do we need this? There's a `Disposable` class in `System.Reactive` that we use in other places.